microsoft-edge/devtools-guide-chromium/css/inspect.md

```diff
@@ -0,0 +1,82 @@
+
+## Getting element information from the Inspect overlay
+
+When the **Inspect** tool is active, hovering over any element on the rendered webpage displays the **Inspect** overlay. This overlay displays general and accessibility information about that element.  
+
+:::image type="complex" source="images/inspect-tool-padding-margin.msft.png" alt-text="The inspector overlay showing the information of a heading in the browser":::
+   The inspector overlay showing the information of a heading in the browser
+:::image-end:::
+
+As you move around the screen, the source view showing the DOM of the document updates and highlights the element you are hovering on in the document tree. 
+
+The **Inspect** overlay shows the following information, depending on the type of element and the styles applied to it:
+
+* The name of the element
+* Its dimensions in pixels
+* Its color as a hexidecimal value and a color swatch
+* Its font settings
+* Its margin and padding in pixels
+
+If the element has been positioned using CSS grid or CSS flexbox you get different icons displayed next to its name.
+
+:::image type="complex" source="images/inspect-tool-grid-element.msft.png" alt-text="An element displayed using CSS grid layout showing an extra icon next to its name in the Inspect overlay" lightbox="images/inspect-tool-grid-element.msft.png":::
+   An element displayed using CSS grid layout showing an extra icon next to its name in the Inspect overlay
+:::image-end:::
+
+:::image type="complex" source="images/inspect-tool-flexbox-element.msft.png" alt-text="An element displayed using CSS flexbox showing an extra icon next to its name in the Inspect overlay" lightbox="images/inspect-tool-flexbox-element.msft.png":::
+   An element displayed using CSS flexbox showing an extra icon next to its name in the Inspect overlay
+:::image-end:::
+
+The **Accessibility** section of the **Inspect** overlay displays information about text-color contrast, the name and the role of the element reported to assistive technoloy and whether the element is keyboard focusable or not. 
+
+For example, in the demo page, the `Bad Contrast` button shows a warning icon next to the contrast value of 1.77. It also shows that the button is not focusable via keyboard as it is a `DIV` element with a class of button instead of a `BUTTON` element.
+
+:::image type="complex" source="images/inspect-tool-bad-contrast.msft.png" alt-text="Elements without sufficient contrast show a warning icon" lightbox="images/inspect-tool-bad-contrast.msft.png":::
+   Elements without sufficient contrast show a warning icon
+:::image-end:::
+
+## Inspecting non-accessible elements
+
+Elements that are covered by other elements or have the CSS property of `pointer-events: hidden` are not available to the **Inspect** tool. In the demo page, try to hover over the `Overlay Button` and you will see that the parent element - in this case `div.wrapper` is shown instead.
```

Comment:
The `Shift` trick doesn't work for elements that are covered by other elements. You can try it on this test page for example:

`data:text/html,<style>div{position:absolute;top:100px;left:100px;width:200px;aspect-ratio:1;background:red;}div+div{background:green;width:300px;}</style><div>1</div><div>2`

You won't be able to select div 1 even when holding shift.

The sentence further down (`If you want to inspect elements like that`) makes you think that holding `Shift` will work for both covered elements and `pointer-events:none` elements, but it won't.

So perhaps rephrase to `Elements that have the CSS style `pointer-events: none` are not available to the **Inspect** tool`.

(note how I also changed `hidden` to `none`, which is the correct CSS value).

Reply:
fixed